packages/ui-shared/src/components/PageTitle/PageTitle.tsx

```diff
@@ -23,24 +25,57 @@ type Props = {
 };
 
 const cn = cnCreate('mfui-beta-page-title');
-const PageTitle: React.FC<Props> = ({ title, breadcrumbs, badge, className, classes = {}, rootRef }) => (
-    <div className={cn([className])} ref={rootRef}>
-        {!!breadcrumbs?.length &&
-            <Breadcrumbs items={breadcrumbs} className={cn('breadcrumbs', {}, classes.breadcrumbs)} />}
-        {badge && <div className={cn('badge')}>{badge}</div>}
-        <Header className={cn('title')} as="h1">{title}</Header>
-    </div>
-);
+const PageTitle: React.FC<Props> = ({
+    title,
+    breadcrumbs,
+    badge,
+    isFullWidth = true,
+    className,
+    classes = {},
+    rootRef,
+}) => {
+    const renderPageTitle = React.useCallback(
+        () => (
+            <>
+                {breadcrumbs?.length &&
+                <Breadcrumbs items={breadcrumbs} className={cn('breadcrumbs', {}, classes.breadcrumbs)} />}
```

Comment:
потерялся отступ внутрь и фигурная скобка на той же строке
```suggestion
                    <Breadcrumbs items={breadcrumbs} className={cn('breadcrumbs', {}, classes.breadcrumbs)} />
                }
```